server/src/main/java/org/opensearch/search/profile/ProfileResult.java

```diff
@@ -86,13 +103,63 @@ public ProfileResult(
         Map<String, Object> debug,
         long nodeTime,
         List<ProfileResult> children
+    ) {
+        this(type, description, breakdown, debug, nodeTime, -1, nodeTime, nodeTime, nodeTime, children, false);
```

Comment:
Should we default the `long maxSliceNodeTime, long minSliceNodeTime, long avgSliceNodeTime` to `-1` as well since we don't really compute them?